chingsley/ireporter
# ch(controller) implement one controller to handle both red-flag and i…

#### What does this PR do?
This PR deals with the implementation of one controller to handle both the red-flag and intervention type records

#### Description of Task to be completed?
- Implement one controller to handle both the red-flag and intervention records
- Delete the individual red-flag and intervention controllers
- Create a new router (recordsRouter.js) in the routes folder to handle both types of records
- Edit the import statement in index.js to import the new router

#### How should this be manually tested?
- Clone the repo
- navigate into the root directory
- run ` npm install ` to install the necessary dependencies
- run ` npm test ` to see the output of the tests
- Or run ` npm start ` and check out the routes (listed in README.md) using POSTMAN
